examples/java/src/main/java/org/apache/beam/examples/common/ExampleUtils.java

```diff
@@ -306,7 +308,7 @@ public void waitToFinish(PipelineResult result) {
         addShutdownHook(jobsToCancel);
       }
       try {
-        job.waitToFinish(-1, TimeUnit.SECONDS, new MonitoringUtil.PrintHandler(System.out));
```

Comment:
logging will be configured through DataflowPipelineOptions.getJobMessageOutput(). It defaults to System.out.

The logging in examples will be better, since it will be configurable.